### PR TITLE
Add link to Issue #34202 in udp docs

### PR DESCRIPTION
**Pull request by wcampbell0x2a**

### CODE REVIEW

File: library/std/src/net/udp.rs [hunk 1]
```diff
@@ -159,7 +159,7 @@ impl UdpSocket {
     /// This will return an error when the IP version of the local socket
     /// does not match that returned from [`ToSocketAddrs`].
     ///
-    /// See issue #34202 for more details.
+    /// See [`Issue #34202`] for more details.
```
Comment (m-ou-se) [inline library/std/src/net/udp.rs:162]: The backticks are only needed for code, to format it monospace font.
```suggestion
    /// See [issue #34202] for more details.
```

File: library/std/src/net/udp.rs [hunk 2]
```diff
@@ -169,6 +169,8 @@ impl UdpSocket {
     /// let socket = UdpSocket::bind("127.0.0.1:34254").expect("couldn't bind to address");
     /// socket.send_to(&[0; 10], "127.0.0.1:4242").expect("couldn't send data");
     /// ```
+    ///
+    /// [`Issue #34202`]: https://github.com/rust-lang/rust/issues/34202
```
Comment (m-ou-se) [inline library/std/src/net/udp.rs:173]:
```suggestion
    /// [issue #34202]: https://github.com/rust-lang/rust/issues/34202
```

File: library/std/src/net/udp.rs (hunk 1 above)
Comment (wcampbell0x2a) [reply library/std/src/net/udp.rs]: fixed

File: library/std/src/net/udp.rs (hunk 2 above)
Comment (wcampbell0x2a) [reply library/std/src/net/udp.rs]: fixed

**Comment by m-ou-se** [comment]
Thanks!

@bors r+ rollup

**Pull request merged**